dts/bindings/spi/st,stm32h7-spi.yaml

```diff
@@ -32,3 +32,9 @@ properties:
   fifo-enable:
     type: boolean
     description: Enable the SPI FIFO usage for performance improvement.
+
+  limited-instance:
+    type: boolean
+    description: |
+      Indicates if the SPI instance is a limited-features instance.
```

Comment:
Does it comes with other limitations ?
Otherwise I think we should somehow explicit that this is a data width limitation.